### PR TITLE
docs(cdk/dialog): add initial setup instructions for CDK dialogs

### PR DESCRIPTION
**Pull request by ifrntnd**
Added instructions for including structural styles required by CDK dialogs. Noted that without importing `@angular/cdk/overlay-prebuilt.css` in the global stylesheet, CDK dialogs did not work correctly.

### CODE REVIEW

File: src/cdk/dialog/dialog.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 The `Dialog` service can be used to open unstyled modal dialogs and to build your own dialog
 services.
 
+### Initial setup
+The CDK dialogs depend on a small set of structural styles to work correctly. If you're using
+Angular Material, these styles have been included together with the theme, otherwise if you're
+using the CDK on its own, you'll have to include the styles yourself. You can do so by importing
+the prebuilt styles in your global stylesheet:
```
Comment (crisbeto) [inline src/cdk/dialog/dialog.md:8]: Might be worth mentioning that one can also get these styles by including the `cdk.overlay` mixin.

Comment (ifrntnd) [reply src/cdk/dialog/dialog.md:8]: @crisbeto I've updated the instructions to include an alternative method using the cdk.overlay mixin in Sass.

**Review by crisbeto — APPROVED** [review]

**Comment by crisbeto** [comment]
@ifrntnd I can't merge in the change because of the third merge commit. Can you drop it from the PR?

**Comment by ifrntnd** [comment]
> @ifrntnd I can't merge in the change because of the third merge commit. Can you drop it from the PR?

Done. I dropped the third merge commit from the PR

**Review by crisbeto — APPROVED** [review]

**Pull request merged**